docs/whats-new.asciidoc

```diff
@@ -4,28 +4,175 @@
+Other versions: {security-guide-all}/8.17/whats-new.html[8.17] | {security-guide-all}/8.16/whats-new.html[8.16] | {security-guide-all}/8.15/whats-new.html[8.15] | {security-guide-all}/8.14/whats-new.html[8.14] | {security-guide-all}/8.13/whats-new.html[8.13] | {security-guide-all}/8.12/whats-new.html[8.12] | {security-guide-all}/8.11/whats-new.html[8.11] | {security-guide-all}/8.10/whats-new.html[8.10] | {security-guide-all}/8.9/whats-new.html[8.9] | {security-guide-all}/8.8/whats-new.html[8.8] | {security-guide-all}/8.7/whats-new.html[8.7] | {security-guide-all}/8.6/whats-new.html[8.6] | {security-guide-all}/8.5/whats-new.html[8.5] | {security-guide-all}/8.4/whats-new.html[8.4] | {security-guide-all}/8.3/whats-new.html[8.3] | {security-guide-all}/8.2/whats-new.html[8.2] | {security-guide-all}/8.1/whats-new.html[8.1] | {security-guide-all}/8.0/whats-new.html[8.0] | {security-guide-all}/7.17/whats-new.html[7.17] | {security-guide-all}/7.16/whats-new.html[7.16] | {security-guide-all}/7.15/whats-new.html[7.15] | {security-guide-all}/7.14/whats-new.html[7.14] | {security-guide-all}/7.13/whats-new.html[7.13] | {security-guide-all}/7.12/whats-new.html[7.12] | {security-guide-all}/7.11/whats-new.html[7.11] | {security-guide-all}/7.10/whats-new.html[7.10] |
 {security-guide-all}/7.9/whats-new.html[7.9]
 
 // NOTE: The notable-highlights tagged regions are re-used in the Installation and Upgrade Guide. Full URL links are required in tagged regions.
 // tag::notable-highlights[]
 
+
+[float]
+== Generative AI enhancements
+
+[float]
+=== Automatically migrate Splunk SIEM rules
+
+{security-guide}/siem-migration.html[Automatic Migration] for detection rules helps you quickly convert SIEM rules from the Splunk Processing Language (SPL) to the Elasticsearch Query Language ({esql}). If comparable Elastic-authored rules exist, it simplifies onboarding by mapping your rules to them. Otherwise, it creates custom rules on the fly so you can verify and edit them instead of writing them from scratch.
+
+[role="screenshot"]
+image::whats-new/images/8.18/security-siem-migration-1.png[The Upload Splunk SIEM rules flyout]
+
+[float]
+=== Automatic Import improvements
+
+{security-guide}/automatic-import.html[Automatic Import] now allows you to select API (CEL input) as a data source and to provide the associated OpenAPI specification (OAS) file to automatically generate a CEL program to consume an API.
+
+[float]
+=== Control which alerts Attack Discovery analyzes 
+
+You can now specify which alerts {security-guide}/attack-discovery.html[Attack Discovery] analyzes using a date and time selector and a KQL filter.
+
+[role="screenshot"]
+image::whats-new/images/8.18/security-attack-discovery-settings.png[Attack Discovery's settings menu,60%]
+
+[float]
+=== Citations and documentation for AI Assistant 
```

Comment:
I would probably flip the title to start with a verb to make it more direct. This is the pattern for other headings too.